recipe/bld.bat

```diff
@@ -1,93 +1,76 @@
-@echo off
+@echo ON
 
-mkdir build
-cd build
-
-set CMAKE_CONFIG="Release"
-
-:: Set the right msvc version according to Python version
-REM write a temporary batch file to map cl.exe version to visual studio version
-echo @echo 15=9 > msvc_versions.bat
-echo @echo 16=10 >> msvc_versions.bat
-echo @echo 17=11 >> msvc_versions.bat
-echo @echo 18=12 >> msvc_versions.bat
-echo @echo 19=14 >> msvc_versions.bat
+git clone https://github.com/Itseez/opencv_contrib --single-branch --branch %PKG_VERSION% --depth 1
```

Comment:
So, I guess we want to install the development modules. I'm ok with that. Could we try downloading a tarball instead?